Civi/Api4/Utils/FormattingUtil.php

```diff
@@ -115,6 +115,10 @@ public static function formatInputValue(&$value, ?string $fieldName, array $fiel
     if ($suffix) {
       $options = self::getPseudoconstantList($fieldSpec, $fieldName, $params, $operator ? 'get' : 'create');
       $value = self::replacePseudoconstant($options, $value, TRUE);
+      // If all the pseudoconstant lookups fail, we want to return no API results, not an exception.
+      if (!$value) {
+        $value[0] = FALSE;
```

Comment:
I think this should be 
```suggestion
        $value = FALSE;
```

Reply:
This doesn't fix the problem.  `CRM_Core_DAO::createSQLFilter()` expects a string if the operator is `=` and an array if the operator is `IN`.  There's no one format that will satisfy both, so I moved the fix to the place where operator-specific SQL clause modifications happen.